src/templates/next/layouts/Homepage/Homepage.tsx

```diff
@@ -10,9 +10,13 @@ const HomepageLayout = ({
 }: HomePageSchema) => {
   return (
     <Skeleton site={site} page={page}>
-      {content.map((component) =>
-        renderComponent({ component, LinkComponent }),
-      )}
+      <div
+        className={`[&_.component-content]:max-w-[1240px] [&_.component-content]:px-6 [&_.component-content]:md:px-10 [&_.component-content]:mx-auto`}
```

Comment:
nit: can add a comment here about its relationship with the `ComponentContent` constant and the reason why we are not using it directly? Just in case we forget in the future.

Reply:
[319bacd](https://github.com/isomerpages/isomer-components/pull/52/commits/319bacd6a2cc9e1440fc098548305161343f8ddf)